src/components/manage/Controlpanels/Controlpanel.jsx

```diff
@@ -43,6 +44,41 @@ const messages = defineMessages({
   },
 });
 
+// Filters props.controlpanel.schema to only valid/relevant fields
+const filterSchema = (controlpanel) => {
+  const panelType = controlpanel['@id'].substring(
+    controlpanel['@id'].lastIndexOf('/') + 1,
+  );
```

Comment:
@danalvrz You could do
```suggestion
  const panelType = controlpanel['@id'].split('/').pop();
```